beekeeper-path-scheduler-apiary/src/main/java/com/expediagroup/beekeeper/scheduler/apiary/messaging/FilteringMessageReader.java

```diff
@@ -53,8 +53,20 @@ private boolean isFiltered(MessageEvent messageEvent) {
     if (listenerEvent == null) {
       return true;
     }
-    return filters.stream()
-        .anyMatch(filter -> filter.filter(listenerEvent));
+
+    Boolean shouldFilter = false;
+
+    for(ListenerEventFilter f : filters) {
```

Comment:
```suggestion
    for(ListenerEventFilter filter : filters) {
```

Reply:
This was some leftover debug logic - removed & reverted to what was there previously

Reply:
This was some leftover debug logic - removed & reverted to what was there previously